### ISSUE TITLE
Reported frequency of messages changes when bag is paused

### ISSUE THREAD

**Issue by peci1** [xqms/rosbag_fancy#20]
When playing a bag with `rosbag_fancy play --clock` and `/use_sim_time` set to true, when I pause the bag, the frequency goes slowly down. This should not happen IMO as the interesting thing is sim-time frequency of messages and not real-time frequency.

**Comment by xqms** [comment]
That is true - if I remember correctly, all displayed rates are based on wall time, not ROS time. Back then I designed the system for recording, not playback. In that case it seemed more relevant to report "real" frequencies, and, more importantly, bandwidths for analysis of the recording situation.

I guess especially in the playback situation `ros::Time` is more applicable.

I don't have the bandwidth to work on this in the near time. But PRs are welcome!
